packages/common/src/multichain/models.ts

```diff
@@ -8,10 +8,10 @@ import {RunnerQueryBaseModel} from '../project';
 
 export class MultichainProjectManifestModel implements MultichainProjectManifest {
   @Equals('1.0.0')
-  specVersion: string;
+  specVersion!: string;
   @IsString({each: true})
-  projects: string[];
+  projects!: string[];
   @IsObject()
   @Type(() => RunnerQueryBaseModel)
-  query: QuerySpec;
+  query!: QuerySpec;
```

Comment:
Can you please explain this, it doesn't make sense to me to need the `!`. Same applies to the other files with this change

Reply:
How should this be modified? The code only declares a class but doesn’t use it anywhere. Should I define a constructor for it?

Reply:
This is used by class-transformer/class-validator so it will construct this class and validate it. It might be worth looking at their repos to see how others have solved this